src/core_modules/capture-core-utils/rulesEngine/processors/rulesEffectsProcessor/rulesEffectsProcessor.js

```diff
@@ -97,8 +99,17 @@ export function getRulesEffectsProcessor(
         let outputValue;
         if (normalizedValue || normalizedValue === 0 || normalizedValue === false) {
             const converterName: string = mapTypeToInterfaceFnName[valueType];
+            if (!converterName) {
+                log.warn(errorCreator('converter for valueType is missing')({ valueType }));
+                return valueType;
+            }
             // $FlowExpectedError
-            outputValue = outputConverters[converterName](normalizedValue);
+            const outputConverter = outputConverters[converterName];
+            if (!outputConverter) {
+                log.warn(errorCreator('outputConverter for valueType is missing')({ valueType }));
+                return valueType;
+            }
+            outputValue = outputConverter(normalizedValue);
```

Comment:
I think it makes sense log something here if we don't find the converterName or outputConverter, so that's good. But I think returning `''` would be preferable (instead of valueType). Also, maybe we can combine the logging?

Anyway, from what I could gather the main problem here is that we're missing valueType for the dataElements that are passed into the rules engine (i.e, valueType here will always be undefined for all dataElements). Great if you can look into this.